source/releasenotes/2023-09-01-early-access-customer-scheduled-jobs.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Early Access: Customer Scheduled Jobs"
+published_date: "2023-09-01"
+categories: [new-feature, tools-apis]
```

Comment:
And let's put the category description in the JSON objects too. That'll be nice to put in the meta description of the category pages.

Reply:
cc @ingridkwok, can you confirm we should use the category descriptions?

Reply:
@stevector should I put much thought into the color attribute per category in this file? I assume I shouldn't give it much thought, since there's still the design work to do which will determine whether any categories require visual distinction

Reply:
@stevector done! https://github.com/pantheon-systems/documentation/pull/8813/commits/8a84152c563d743c97de3bba61315d403ee0886d 

I added a few comments for the last 3 which is worth calling out. I think we should:

* Delete platform category, use infrastructure instead (Added as a comment in the google worksheet and tagged Ingrid for review)
* Delete integration category, use tools-apis instead (Added as a comment in the google worksheet and tagged Ingrid for review)
* Singal review needed from Ingrid and Design team for a new category "Documentation" (Added to the google worksheet in the last row)

Reply:
Just realized you asked me to populate all _used_ categories and instead I populated all _defined_ categories. Let me know if you want me to delete the ones not yet used by any of the backfill or if what I pushed is okay